src/devices/video/huc6270.h

```diff
@@ -4,6 +4,67 @@
 
     Hudson/NEC HuC6270 interface
 
+    Pinouts, QFP64
+
+                                  )
+                                  D
+                                  N
+                                  G
+                                  (
+                            5 4 3 3 2 1 0         3
+                5 4 3 2 1 0 1 1 1 s 1 1 1 9 8 7 6 d 5 4 3 2 1 0
+                A A A A A A D D D s D D D D D D D d D D D D D D
+                M M M M M M M M M V M M M M M M M V M M M M M M
+                | | | | | | | | | | | | | | | | | | | | | | | |
+               /-----------------------------------------------\
+          MA6 -|                                               |-/MRD
+          MA7 -|                                               |-/MWR
+          MA8 -|                                               |-VD0
+          MA9 -|                                               |-VD1
+         MA10 -|                                               |-VD2
+         MA11 -|                                               |-VD3
+    Vss4(GND) -|                                               |-VD4
+         Vdd4 -|                    HuC6270                    |-Vss2(GND)
+         MA12 -|                                               |-Vdd2
+         MA13 -|                                               |-VD5
+         MA14 -|                                               |-VD6
+         MA15 -|                                               |-VD7
+         /IRQ -|                                               |-SP8G
+        /BUSY -|                                               |-DISP
+           A0 -|                                               |-/HSYN
+           A1 -|                                               |-/VSYN
+               \-----------------------------------------------/
+                | | | | | | | | | | | | | | | | | | | | | | | |
+                / / / D D D D D D D V D D D D D D V D D D 8 C /
+                C W R 1 1 1 1 1 1 9 s 8 7 6 5 4 3 d 2 1 0 / K R
+                S R D 5 4 3 2 1 0   s             d       1   E
+                                    1             1       6   S
+                                    (                         E
+                                    G                         T
+                                    N
+                                    D
+                                    )
+
+    CK: Input clock
+    /RESET: Reset pin
+    /CS: Chip select
+    /WR: Write enable
+    /RD: Read enable
+    /IRQ: Interrupt output to host CPU
+    /BUSY: Busy flag
+    /MRD: Memory read enable
+    /MWR: Memory write enable
+    /HSYN: Horizontal sync
+    /VSYN: Vertical sync
+    MA0-15: Memory address
+    MD0-15: Memory data
+    VD0-7: Pixel data output
+    SP8G: Sprite/Background flag, Also highest bit of pixel data output
+    A0-1: Host interface address
+    D0-15: Host interface data
+    8/16: 8 or 16 bit host interface mode?
+    DISP: Unknown, display related?
```

Comment:
According to HuC6270 manual DISP pin is programmable and tied to video sync signals, presumably having sense in external genlocking cfr. page _H7-6_.